apis/beacon/states/validator_balances.yaml

```diff
@@ -90,8 +90,11 @@ post:
       in: path
       $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
   requestBody:
-    description: "An array of either hex encoded public key (any bytes48 with 0x prefix) or validator index"
-    required: false
+    description: |
+      Either hex encoded public key (any bytes48 with 0x prefix) or validator index.
+
+      If the supplied list is empty (i.e. the body is `[]`) then balances will be returned for all validators.
```

Comment:
Shall we update to `(i.e. the body is [] or the body is empty)` given the discussion in the comments? I don't mind leaving it undefined unless people are going to rely on that behaviour, but it does seem cleaner than an error...

Reply:
in favor of this, also need to make sure body is not required by the schema

Reply:
I'll update accordingly.